### PR TITLE
cleanup(RBAC): RHICOMPL-3067 remove Compliance reporter role

### PR DESCRIPTION
**Pull request by romanblanco**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -113,7 +113,8 @@ services:
       - DATABASE_USER=insights
       - DATABASE_PASSWORD=insights
       - API_PATH_PREFIX=/api/rbac
-      - DEVELOPMENT=True
+      - DEVELOPMENT=False
+      - BYPASS_BOP_VERIFICATION=True
```
Comment (marleystipich2) [inline docker-compose.yml:117]: What's the purpose of this change? Just curious

**Review by marleystipich2 — COMMENTED** [review]
These updates look great to me! I just had that one question about one of the changes.

File: docker-compose.yml (hunk 1 above)
Comment (skateman) [reply docker-compose.yml:117]: I'm also curious :thinking:

Comment (romanblanco) [reply docker-compose.yml:117]: We need this to be able to have a user that is not _Org. Administrator_ in local development. 

The `DEVELOPMENT=False` is to pass an explicit identity. Otherwise, [dev_middleware.py](https://github.com/RedHatInsights/insights-rbac/blob/0200aac03ee2064fdad9f11cf885a8264b12db27/rbac/rbac/dev_middleware.py#L33) could alter it. _(see. [relevant Slack thread](https://ansible.slack.com/archives/CE6Q8FR99/p1652210324932339?thread_ts=1652105183.338989&cid=CE6Q8FR99))_

**Review by skateman — APPROVED** [review]
The Seal of Approval
